### PR TITLE
Neat 68 update information architect text

### PR DESCRIPTION
**Pull request by nikokaoja**
- updated mock graph step
- Finilize Info Architect text
- Update text up to DMS Architect
- Created videos and uploaded them to youtube
- Added videos to text up to DMS Architect

UI improvements:
- enabled right click 
- workflow setting are saved prior workflow start
- filtering added on rules to show only XLSX files
- removed bunch of legacy `console.logs`

### CODE REVIEW

File: cognite/neat/app/ui/neat-app/src/views/WorkflowView.tsx [hunk 1]
```diff
@@ -565,7 +565,7 @@ return (
               </Panel>
             </ReactFlow>
 
-            <Button variant="outlined" onClick={startWorkflow} sx={{ marginTop: 2, marginRight: 1 }}>Start workflow</Button>
+            <Button variant="outlined" onClick={ () => {saveWorkflow(); startWorkflow()} } sx={{ marginTop: 2, marginRight: 1 }}>Start workflow</Button>
```
Comment (doctrino) [inline cognite/neat/app/ui/neat-app/src/views/WorkflowView.tsx:568]: Sweet!

File: docs/data-modeling-lifecycle/tutorial-1-knowledge-acquisition.md [hunk 2]
```diff
@@ -14,19 +14,19 @@ units are partially overlapping, both in concepts and data. Lots of the value in
...
-`neat` has been designed to facilitate this process by providing a way to iterate on and developed this model.
+The expert elicitation is the process of distilling the domain expert knowledge and turning it into a shared knowledge artifact such as an Enterprise Data Model (covering the entire suite of use-cases and domains and business units).
+`neat` has been designed to facilitate this process by providing a way to iterate on and developed the enterprise data model.
 
 ## Use Case
 
-In this tutorial, we will focus on the Power & Utilities industry. We will have two domain experts, one that
-focuses on wind farm operation and one that focuses on grid analysis, lets call them Jon and Emma. In addition,
-we will have an information architect, let's call him David, who will be responsible for combining the
+In this tutorial, we will focus on the Power & Utilities industry, and try to build power to consumer data model.
```
Comment (doctrino) [inline docs/data-modeling-lifecycle/tutorial-1-knowledge-acquisition.md:22]:
```suggestion
In this tutorial, we will focus on the Power & Utilities industry and try to build power to consumer data model.
```

File: docs/data-modeling-lifecycle/tutorial-1-knowledge-acquisition.md [hunk 3]
```diff
@@ -89,25 +90,16 @@ When Jon has defined all the statements, he can validate the sheet using `neat`.
...
-and warnings.
-
-<img src="images/run_workflow.png" height="300">
 
 ### Summary
 
 **Domain Expert Task.**
 
-1. (Required) Gathering statements in a spreadsheet.
+1. (Required) Gathering statements in a spreadsheet.****
```
Comment (doctrino) [inline docs/data-modeling-lifecycle/tutorial-1-knowledge-acquisition.md:102]:
```suggestion
1. (Required) Gathering statements in a spreadsheet.
```

File: docs/data-modeling-lifecycle/tutorial-1-knowledge-acquisition.md [hunk 4]
```diff
@@ -315,62 +334,95 @@ The enterprise data model is now ready to be validated in `neat`. David will val
...
-### Extending the Sheet
+- `RulesToSemanticDataModel` : which will convert `Rules` into semantic data model and store it as a Turtle file
+- `GraphStoreConfiguration`: which will configure the internal RDF store
+- `GraphFromRdfFile` : which will load the semantic data model (the Turtle file) into the internal RDF store
 
-David will also add two new columns to the `Properties` sheet. The first column is called `Source`, and it is used
-to specify where the statement comes from, or what standard that matches the statement. The second column is called
-`MatchType`, and tells whether the source is partially or fully matching the statement.
+The process of configuration and execution of these steps is shown in the video below:
+https://youtu.be/_iTQatLBpns
```
Comment (doctrino) [inline docs/data-modeling-lifecycle/tutorial-1-knowledge-acquisition.md:359]: Should this also be put into an `iframe`?

**Review by doctrino — APPROVED** [review]
Way to go! Love the extra effort with youtube videos :)

File: cognite/neat/app/ui/neat-app/src/views/WorkflowView.tsx (hunk 1 above)
Comment (nikokaoja) [reply cognite/neat/app/ui/neat-app/src/views/WorkflowView.tsx:568]: it was easier than expected, but took a while to find out where and how to change this

File: docs/data-modeling-lifecycle/tutorial-1-knowledge-acquisition.md (hunk 4 above)
Comment (nikokaoja) [reply docs/data-modeling-lifecycle/tutorial-1-knowledge-acquisition.md:359]: it should not be there at all :D

**Pull request merged**